code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -559,6 +559,16 @@ Code:
 		if (54) // Beepsky, Medibot, Floorbot, and Cleanbot access
 			menu = "<h4>[PDAIMG(medbot)] Bots Interlink</h4>"
 			bot_control()
+		if (55) // Emoji Guidebook for mimes
+			menu = "<h4>[PDAIMG(emoji)] Emoji Guidebook</h4>"

```

Comment:
i know i got you maintainers kinda fliky with this pr but don't merge it yet because this is brooo ho ken i mean wow it does not work at all

Reply:
got it working